<launch>
    <node pkg="local_map" type="local_map_pub" output="screen" name="local_map_pub">  
        <param name="localmap_x_size" value="14"/>  
        <param name="localmap_y_size" value="14"/>
        <param name="resolution" value="0.1"/>
        <param name="DBSCAN_R" value="5.0"/>
        <param name="DBSCAN_N" value="4"/>
    </node>
</launch>
